index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="toolbar-container">
+      <a href=""
+        ><img id="logo" src="./images/header/My Logo.png" alt="logo"
+      /></a>
+      <a href=""
+        ><img
+          id="nav-logo"
+          src="./images/header/Hamburger_icon.svg.png"
+          alt="logo"
+      /></a>
```

Comment:
- [ ] Following the best practices for semantic tags, this is your website nav bar and it should be in a `nav` tag and not an `a` tag. I'll suggest you update this :+1: :100: 
- [ ] Also currently, this hamburger icon is not visible in the browser, this is due to the max-width media query you used, (I will point out the media query issue later in this review, check my other comments). I suggest you fix this also. :smile: 

- - YOUR CURRENT DESIGN:
![Screenshot from 2023-06-02 17-55-38](https://github.com/mhassa204/Portfolio-Project/assets/107632780/85a4180f-0130-41d5-ad05-ab84b2911bf3)

- - FIGMA DESIGN:
![Screenshot from 2023-06-02 17-56-16](https://github.com/mhassa204/Portfolio-Project/assets/107632780/97e9b1d4-1005-4955-9709-6cb8f2e466fc)